src/components/Settings/SettingsJellyfin.tsx

```
@@ -108,61 +147,122 @@ const SettingsJellyfin: React.FC<SettingsJellyfinProps> = ({
-      .nullable()
-      .test('valid-url', intl.formatMessage(messages.validationUrl), isValidURL)
-      .test(
-        'no-trailing-slash',
-        intl.formatMessage(messages.validationUrlTrailingSlash),
-        (value) => !value || !value.endsWith('/')
-      ),
-  });
+        .required(intl.formatMessage(messages.validationHostnameRequired)),
+      port: Yup.number().when(['hostname'], {
+        is: (value: unknown) => !!value,
+        then: Yup.number()
```

Biome (2.4.6)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)